src/validators/fieldset-validators/fieldset-validators.ts

```diff
@@ -0,0 +1,70 @@
+import { Validator } from "../validator";
+
+export const requiredFieldset: Validator<string> = {
+    validate: (id: string) => {
+        const el = document.querySelector(`gcds-fieldset[fieldset-id=${id}]`);
+        const elChildren = el.children;
+
+        let isValid = validateFieldsetElements(el, elChildren);
+
+        return isValid.includes(false) ? false : true;
```

Comment:
Could just "`return !isValid.includes(false)`". No need for the "`? false : true;`"